src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
+                    ]
+                ]
+
+        actionDetail =
+            let
+                proofCode =
+                    case claim.proofCode of
+                        Just code ->
+                            code
+
+                        Nothing ->
+                            ""
+            in
+            div
+                [ class "block mt-6" ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.action") ]
+                , p [ class "text-left mt-2 text-lg w-full" ] [ text claim.action.description ]
+                , case claim.proofPhoto of
+                    Just url ->
+                        div
+                            [ class "relative h-auto w-auto text-center text-white"
+                            , Utils.onClickNoBubble (OpenPhotoModal claim)
+                            ]
+                            [ div
+                                [ class "z-10 absolute bottom-1 left-1 bg-black bg-opacity-60 p-4" ]
+                                [ p [ class "text-xs text-left w-full uppercase" ] [ text (t "community.actions.form.verification_code") ]
+                                , p [ class "text-base font-bold font-normal text-left w-full" ] [ text proofCode ]
+                                ]
+                            , div []
+                                [ img
+                                    [ class "z-0 rounded-md my-4 w-full"
+                                    , style "height" "200px"
+                                    , src url
+                                    ]
+                                    []
+                                ]
+                            ]
+
+                    Nothing ->
+                        text ""
+                ]
+
+        header =
+            div [ class "flex mt-16 space-x-8 justify-center" ]
+                [ photoAndTagName
+                , claimDateAndState
+                ]
+
+        body =
+            div
+                [ class "block" ]
+                [ rewardInfo
+                , approvedBy
+                , disapprovedBy
+                , pendingVote
+                , actionDetail
+                ]
+
+        footer =
+            if isVotable claim loggedIn.accountName loggedIn.shared.now then
+                div [ class "flex justify-between space-x-4 mt-8 object-bottom" ]
+                    [ button
+                        [ class "button button-danger"
+                        , onClick (OpenVoteModal claim.id False)
+                        ]
+                        [ text (t "dashboard.reject") ]
+                    , button
+                        [ class "button button-primary"
+                        , onClick (OpenVoteModal claim.id True)
+                        ]
+                        [ text (t "dashboard.verify") ]
+                    ]
+
+            else
+                div [] []
```

Comment:
Using `text ""` is better than `div [] []`, because `text ""` doesn't add anything to the HTML generated, and `div [] []` adds an empty `div` tag